pgwire-server/src/main/clojure/xtdb/pgwire.clj

```diff
@@ -433,10 +383,9 @@
     :cols [{:column-name "transaction_isolation" :column-oid oid-varchar}]
     :rows [["read committed"]]}
 
-   ;; ODBC issues this query by default, you may be able to disable with an option
-   {:q "select oid, typbasetype from pg_type where typname = 'lo'"
-    :cols [{:column-name "oid", :column-oid oid-varchar} {:column-name "typebasetype", :column-oid oid-varchar}]
-    :rows []}
+   {:q "show timezone"
```

Comment:
this can probably go into the grammar now, and be supported as a proper query rather than a hard coded string

Reply:
Yes, although didn't want to burden this PR with said changes. Will spin out these plus other tooling/postgres dialect related features into their own cards.